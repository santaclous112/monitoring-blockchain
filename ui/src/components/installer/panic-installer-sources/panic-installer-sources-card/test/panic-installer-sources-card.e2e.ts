import { newE2EPage } from '@stencil/core/testing';

describe('panic-installer-sources-card', () => {
  it('renders', async () => {
    const page = await newE2EPage();
    await page.setContent('<panic-installer-sources-card></panic-installer-sources-card>');

    const element = await page.find('panic-installer-sources-card');
    expect(element).toMatchSnapshot();
  });
});
