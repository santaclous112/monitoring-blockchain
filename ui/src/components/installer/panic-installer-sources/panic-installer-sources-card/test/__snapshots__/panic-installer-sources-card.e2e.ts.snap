// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-installer-sources-card renders 1`] = `<panic-installer-sources-card class="hydrated"></panic-installer-sources-card>`;
